api/src/main/resources/workbench.yaml

```diff
@@ -1641,6 +1666,29 @@ definitions:
       approved:
         type: boolean
 
+  ClusterConfig:
+    properties:
```

Comment:
Would it make sense to make these optional with a default? Do we ever see someone wanting more disk without upgrading their machine type, or changing their machine type without upgrading disk?

Reply:
Yes, that's how they work but it wasn't documented here - updated these docs to hopefully make that clear.